.circleci/config.yml

```diff
@@ -48,10 +48,10 @@ commands:
       - persist_to_workspace:
           root: *working_directory
           paths:
-            - packages/**/dist
-            - playground/dist
-            - visual-testing-app/dist
-            - application-templates/**/dist
+            - packages
```

Comment:
This was premature. packages (e.g. application-shell) sometimes contain things from src they would actually like to reference and publish. For application-shell this is test-utils.